src/pages/home/base/mobile-core/configuration/index.md

```diff
@@ -74,14 +74,17 @@ For implementation details, please refer to [Configuration API reference](./api-
 
 ## Using a bundled file configuration
 
-You can include a bundled JSON configuration file in your app package to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](#configure-with-app-id-per-environment) approach.
+Applications which need to get data from the SDK early in the application lifecycle should use a bundled file configuration. This will allow the SDK to not be at the mercy of the network connection for downloading the remote configuration, and will prioritize the bundled configuration in early, app launch scenarios. If you are going to use a bundled file configuration to help with early app processing, it is strongly recommended that you also use [bundled rules](../rules-engine/index.md#using-bundled-rules).
```

Comment:
I think saying "being at the mercy of the network connection" is a little bit too colloquial for our docs. I'd go with something like this:

"This removes the dependency of the network connection for downloading..."

Additionally, I'm not a huge fan of using "it is recommended...", so let's make it as follows instead:

"... with early app processing, you should also use bundled rules."

Reply:
I agree we can improve the wording, but "this removes the dependency of the network connection" isn't  totally accurate.

How about: "This will allow the SDK to properly process events before a remote configuration is downloaded, using the bundled configuration in early, app launch scenarios."

Reply:
I like that! That reads good to me.